<launch>
    <node pkg="LAIS" type="fj005" name="fj005" output="screen">
        <param name="basic_config" type="string" value="$(find LAIS)/config/basic_config.yaml" />
        <param name="force_start" type="bool" value="false" />
    </node>
    <node pkg="LAIS" type="camera" name="camera" />


</launch>
